.github/workflows/ci.yml

```diff
@@ -642,6 +642,40 @@ jobs:
           path: diagnostics*.gz
           retention-days: 1
 
+  benchmarks:
```

Comment:
this should be labeled as `unit-benchmark` or something like to indicate this doesn't perform integration or cluster based benchmark